### PR TITLE
Fix/unified ui integration tests

### PR DESCRIPTION
**Pull request by philipnrmn**
This PR fixes tests which were broken after the `unified-ui` branch was merged into master. This was mostly due to the sidebar being replaced by the task detail page. 

Two files have been removed, one which tested the (no-longer used) overlay header, and one which tested the task side panel. The functionality which the latter protected will be reintroduced by the task detail page PR #118 .

Note also that one test suite is still failing, but these are appropriate failures which were not caused by the unified-ui merge and thus not fixed in this PR.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by rcorral** [comment]
@philipnrmn PR #118 only adds one test for the Task detail page, but we actually have 3 different test cases here. Can we have that PR introduce all 3?

File: tests/pages/system/overview/repositories/RepositoriesTab-cy.js [hunk 1]
```diff
@@ -61,7 +61,10 @@ describe('Installed Packages Tab', function () {
       .should('contain', 'Remove Repository');
   });
 
-  it('closes modal after uninstall is successful', function () {
+  xit('closes modal after uninstall is successful', function () {
```
Comment (orlandohohmeier) [inline tests/pages/system/overview/repositories/RepositoriesTab-cy.js]: Let's not "x" this one...

**Comment by jfurrow** [comment]
Great, @philipnrmn! I'll give it my label but I am also wondering about the `xit` test that @orlandohohmeier mentioned.

File: tests/regressions/OverlayHeaderHeight-cy.js [hunk 2]
```diff
@@ -1,45 +0,0 @@
-describe('Overlay Header Height [00j]', function () {
```
Comment (orlandohohmeier) [inline tests/regressions/OverlayHeaderHeight-cy.js]: Is this test really outdated? Is there no head bar any more? @rcorral @philipnrmn

**Comment by orlandohohmeier** [comment]
@jfurrow I dropped that commit, so that we should be fine to merge it after addressing https://github.com/dcos/dcos-ui/pull/174/files#r63268568

File: tests/regressions/OverlayHeaderHeight-cy.js (hunk 2 above)
Comment (orlandohohmeier) [inline tests/regressions/OverlayHeaderHeight-cy.js]: I've adjusted the respective commit, to not delete this file, as we will reintroduce the header bar.

**Comment by orlandohohmeier** [comment]
I've rebased the branch and addressed all the issues. Let's :ship:

**Comment by orlandohohmeier** [comment]
@rcorral Let's (re)implement the task detail test in #118

**Comment by rcorral** [comment]
Let's hold merge until tests are implemented on #118.

**Comment by philipnrmn** [comment]
I'm confused about the need to hold merge - there's no interdependence between the two.

**Comment by pierlo-upitup** [comment]
I see the missing tests added to https://github.com/dcos/dcos-ui/pull/118/commits/02414c996513ce67db3e30c5dab8cac5fad09951.  🚢

**Pull request merged**